.github/workflows/build.yml

```
@@ -0,0 +1,16 @@
+name: build
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.0
```

We should use SHAs for external actions as noted in the SDR

```suggestion
    - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
```